mflowgen/Tile_MemCore/constraints/common.tcl

```diff
@@ -140,14 +140,14 @@ set sb_delay 0.3
 # Use this first command to constrain all feedthrough paths to just the desired SB delay
 set_max_delay -from SB*_IN_* -to SB*_OUT_* [expr ${sb_delay} + ${i_delay} + ${o_delay}]
 # Then override the rest of the paths to be full clock period
-set_max_delay -from SB*_IN_* -to SB*_OUT_* -through [get_pins [list CB*/* DECODE*/* MemCore_inst0*/* FEATURE*/*]] ${dc_clock_period}
+set_max_delay -from SB*_IN_* -to SB*_OUT_* -through [get_pins [list CB*/* DECODE*/* MemCore_inst0*/* FEATURE*/*]] ${clock_period}
 
 #set_input_transition 1 [all_inputs]
 #set_max_transition 10 [all_outputs]
 
 if $::env(PWR_AWARE) {
     source inputs/dc-dont-use-constraints.tcl
-    source inputs/mem-constraints-2.tcl
+    # source inputs/mem-constraints-2.tcl
```

Comment:
Why are these power domains constraints being skipped?

Reply:
These constraints don't work in Genus ... there is no `set_voltage` command for example.

Reply:
Well shouldn't there be some replacement then? Seems like this could replace the power domains flow.

Reply:
The old power domains flow from the previous tapeout that used Genus didn't seem to have anything corresponding to this .. it might be DC-specific. I can double-check with Ankita.